onnx-ecosystem/requirements.txt

```diff
@@ -10,7 +10,7 @@ setuptools==41.0.0
 
 coremltools==2.1
 
-tensorflow==1.15.2
+tensorflow==2.4.0
```

Comment:
some of the tutorials use ```tf.Session()``` which in tf-2.4.0 is no longer available so tutorials would break.
We'd need to change them to ```tf.compat.v1.Session()```